project-directory.md

```diff
@@ -116,90 +116,94 @@ These are all the repos found in the IPFS Github organization. If you still can'
     Examples on how to use go-ipfs.
 
 ## Community Resources
+
+* [awesome-ipfs](https://github.com/ipfs/awesome-ipfs)  
+    Useful resources for using IPFS and building things on top of it.
+
+* [blog](https://github.com/ipfs/blog)  
+    IPFS Blog.
+
 * [community](https://github.com/ipfs/community)  
     General discussion and documentation on community practices.
 
-* [support](https://github.com/ipfs/support)  
-    For questions on how to get IPFS up and running smoothly.
-
 * [infrastructure](https://github.com/ipfs/infrastructure)  
     Tools and systems for the community.
 
+* [logo](https://github.com/ipfs/logo)  
+    The logo for IPFS.
+
+* [meetup-lisbon](https://github.com/ipfs/meetup-lisbon)  
+    The IPFS meetup in Lisbon.
+
 * [pm](https://github.com/ipfs/pm)  
     General project management.
 
+* [support](https://github.com/ipfs/support)  
+    For questions on how to get IPFS up and running smoothly.
+
 * [website](https://github.com/ipfs/website)  
     The source to the IPFS community website at http://ipfs.io.
 
-* [logo](https://github.com/ipfs/logo)  
-    The logo for IPFS.
+## Tools
```

Comment:
I think if we start adding in non-IPFS organization repositories, it might be hard to draw the line, especially for your hundred+ repositories. I'm happy to add them, though.